src/main/java/com/duckblade/osrs/dpscalc/DpsCalcPlugin.java

```diff
@@ -117,6 +115,77 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 			navButton.getOnSelect().run();
 	}
 
+	// subscribe to equipment changes
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
+	{
+		if (!config.enableAutoSync()) return;
+
+		if (event.getContainerId() == InventoryID.EQUIPMENT.getId()) {
+			panel.onEquipmentChanged();
+		}
+	}
+
+	// subscribe to attack style changes
+	@Subscribe
+	public void onVarbitChanged(VarbitChanged event)
+	{
+		if (!config.enableAutoSync()) return;
+
+		if (event.getIndex() == VarPlayer.ATTACK_STYLE.getId()) {
+			panel.onEquipmentChanged();
+		}
+
+		// Unsure which varbit this actually is, or if its even defined anywhere
+		if (event.getIndex() == 83) {
+			panel.onPrayersChanged();
+		}
+	}
+
+	// listen for updates to combat skills
+	@Subscribe
+	public void onStatChanged(StatChanged statChanged)
+	{
+		if (!config.enableAutoSync()) return;
```

Comment:
Have you tested whether these are triggered on drains?

e.g. Draining a single prayer point shouldn't recalculate the entire DPS result. Also, players should be able to distinguish between synchronizing gear and synchronizing stats separately.